### PR TITLE
ROX-16362: Terraform acs-prod-eu-01 on CI

### PR DESCRIPTION
**Pull request by vladbologa**
## Description

Adds acs-prod-eu-01 terraforming to the CD pipeline.

## Checklist (Definition of Done)

- [ ] ~~Unit and integration tests added~~
- [ ] ~~Added test description under `Test manual`~~
- [ ] ~~Documentation added if necessary (i.e. changes to dev setup, test execution, ...)~~
- [x] CI and all relevant tests are passing
- [x] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] ~~Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.~~

## Test manual

No manual testing performed.

### CODE REVIEW

**Comment by stehessel** [comment]
I'd suggest to get https://github.com/stackrox/acs-fleet-manager/pull/915 merged first. @porridge thoughts?

**Comment by kovayur** [comment]
Agree with @stehessel, refactoring PR #913 has been merged.

File: .github/workflows/deploy-production.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ jobs:
     with:
       acs_environment: prod
       github_environment: production
-      deploy_clusters: "acs-prod-dp-01"
+      deploy_clusters: "acs-prod-dp-01 acs-prod-eu-1"
```
Comment (porridge) [inline .github/workflows/deploy-production.yaml:15]: So is it `01` like in the PR title, or `1`?

Comment (vladbologa) [reply .github/workflows/deploy-production.yaml]: It's `acs-prod-eu-01`, thanks for catching this!

**Review by porridge — APPROVED** [review]

**Pull request merged**